src/players/FilePlayer.js

```diff
@@ -49,6 +49,12 @@ export default class FilePlayer extends Base {
     }
   }
   componentWillUnmount () {
+    if (this.hls) {
+      this.hls.destroy()
+    }
+    if (this.dash) {
+      this.dash.reset()
```

Comment:
I wonder if this stuff should move to `stop()` instead? `stop()` is called when unmounting, and we are already doing `hls.detachMedia()` there too.

Reply:
Oh sorry, didn't notice it there. That's what you get when in a hurry. It makes perfect sense, so I'll "fix the fix".